libs/hmpb-interpreter/src/utils/geometry.ts

```diff
@@ -83,11 +83,39 @@ export function median(numbers: ArrayLike<number>): number {
 }
 
 /**
- * https://math.stackexchange.com/a/361419
+ * Compute the inner angle formed by vectors `ba` and `bc`.
+ *
+ * @see https://math.stackexchange.com/a/361419
  */
 export function angleBetweenPoints(a: Point, b: Point, c: Point): number {
-  const abDotBc = (b.x - a.x) * (c.x - b.x) + (b.y - a.y) * (c.y - b.y)
+  const abDotBc = (a.x - b.x) * (c.x - b.x) + (a.y - b.y) * (c.y - b.y)
```

Comment:
why is this changing?

Reply:
I think I made a mistake with my initial transcription from math into JS. I had the direction of the AB vector reversed, so the angles I was getting were the "outer" angle rather than the "inner" angle. That hasn't really mattered yet since I was primarily looking for right angles, but I wanted to make sure I had it right so I was adding tests for it and found this bug.